aws_lambda_builders/workflows/nodejs_npm/workflow.py

```diff
@@ -26,64 +39,110 @@ class NodejsNpmWorkflow(BaseWorkflow):
 
     EXCLUDED_FILES = (".aws-sam", ".git")
 
-    def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=None, osutils=None, **kwargs):
-
-        super(NodejsNpmWorkflow, self).__init__(
-            source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=runtime, **kwargs
-        )
-
-        if osutils is None:
-            osutils = OSUtils()
-
-        subprocess_npm = SubprocessNpm(osutils)
+    CONFIG_PROPERTY = "aws_sam"
 
+    def actions_without_bundler(self, source_dir, artifacts_dir, scratch_dir, manifest_path, osutils, subprocess_npm):
```

Comment:
Doc strings for `actions_without_bundler`, `actions_with_bundler`, and `get_manifest_config`.